ops/repo_init.sh

```diff
@@ -9,5 +9,5 @@ poetry install
 popd
 
 pushd webui
-npm install
+yarn install
```

Comment:
Does this need to install yarn first?

Reply:
same as above - most modern node ships with yarn, but no reason not to do belt and suspenders!